src/bolos/nbgl.c

```diff
@@ -148,10 +148,43 @@ unsigned long sys_nbgl_front_draw_img_file(nbgl_area_t *area, uint8_t *buffer,
 
 unsigned long sys_nbgl_get_font(unsigned int fontId)
 {
-  if (fontId >= STAX_NB_FONTS) {
+  switch (hw_model) {
+  case MODEL_STAX: {
+    unsigned int maxNbFonts;
+    if (sdk_version >= SDK_API_LEVEL_15) {
+      maxNbFonts = STAX_NB_FONTS;
+    } else {
+      maxNbFonts = STAX_NB_FONTS_12;
+    }
+    if (fontId >= maxNbFonts) {
+      return 0;
+    } else {
+      return *((unsigned int *)(STAX_FONTS_ARRAY_ADDR + (4 * fontId)));
+    }
+  }
+  case MODEL_FLEX:
+    fontId -= 11; // BAGL_FONT_INTER_REGULAR_28px
```

Comment:
Would be nice to better explain the reason for this?

Reply:
:shrug: